src/test/java/com/salesforce/dynamodbv2/mt/mappers/CreateTableRequestBuilderTest.java

```diff
@@ -87,11 +87,9 @@ void testExpectedBillingMode(ProvisionedThroughput inputProvisionedThroughput, B
             expectedCapacityUnits = 1L;
         }
 
-        testBuilder.withBillingMode(expectedBillingMode);
-        testBuilder.addSi(indexName, indexType, primaryKey, expectedCapacityUnits);
-        testBuilder.build();
-
         DynamoDbTestUtils.assertExpectedBillingModeIsSet(
-            testBuilder.getCreateTableRequest(), expectedBillingMode, expectedCapacityUnits);
+            testBuilder.withBillingMode(expectedBillingMode)
```

Comment:
for the sake of readablity, consider extracting

```
testBuilder.withBillingMode(expectedBillingMode)
                .addSi(indexName, indexType, primaryKey, expectedCapacityUnits)
                .build()
```
into a variable

Reply:
fixed in 9b96d66e8bd98a44d51568449b06b9449bf4e988